core/controllers/admin.py

```diff
@@ -955,39 +1207,36 @@ def post(self):
 class UpdateUsernameHandler(base.BaseHandler):
     """Handler for renaming usernames."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'old_username': {
+                'schema': {
+                    'type': 'basestring'
```

Comment:
are we expecting this to be Unicode? Also, can you please check with python migration team now whether to use basestring or unicode in most of the cases?

Reply:
As per discussion, [here](https://docs.google.com/document/d/1UN_vC9f3l4HUJDs0WJvWCTbJI5wmrl7AMrjC2nZ8gbo/edit?disco=AAAANLPGtl8), basestring can be used as for now.
Please guide me if I am missing anything.

Reply:
I think that's for the cases where we cannot use unicode* 

But in this case, the existing code verifies the old_username to be a UNICODE string, right?

Reply:
After modification of schema type from basestring to unicode, I again reproduced the same error as mentioned in the doc.